MANIFEST.in

```
@@ -1,3 +1,4 @@
 include README.rst LICENSE test_inflection.py
+graft inflection
```

```suggestion
recursive-include inflection py.typed
```

Just in case a graft picks up bytecode files. I don't think it does, but let's be conservative here.